### ISSUE TITLE
ToUndirected() cannot explicitly specify edge attributes

### ISSUE THREAD

**Issue by jjzhou012** [pyg-team/pytorch_geometric#3879]
### 🐛 Describe the bug

For a graph with one edge, the transform **ToUndirected()** extends its graph label y, which it seems to treat y as edge attribute. Because the dimension of y matches the number of edges, they're both equal to 1.

```python
from torch_geometric.data import Data
from torch_geometric.utils import to_undirected
from torch_geometric.transforms import ToUndirected

x = torch.ones([2,3])
edge_index = torch.tensor([[0], [1]])
edge_attr = torch.tensor([[1,2,3]])
y = torch.tensor([1])

x2 = torch.ones([2,3])
edge_index2 = torch.tensor([[0,0], [1,2]])
edge_attr2 = torch.tensor([[1,2,3], [1,2,3]])
y2 = torch.tensor([1])

trans = ToUndirected()

data = Data(x=x, edge_index=edge_index, y=y, edge_attr=edge_attr)
print('graph with 1 edge', data)
new_data = trans(data)
print('graph with 1 edge, to undirected', new_data)

print('\n')

data2 = Data(x=x2, edge_index=edge_index2, y=y2, edge_attr=edge_attr2)
print('graph with 2 edge', data2)
new_data2 = trans(data2)
print('graph with 2 edge, to undirected', new_data2)
```
Here is the output:
```
graph with 1 edge:  Data(x=[2, 3], edge_index=[2, 1], edge_attr=[1, 3], y=[1])
graph with 1 edge, to undirected:  Data(x=[2, 3], edge_index=[2, 2], edge_attr=[2, 3], y=[2])
# y has been treated as another edge_attr

graph with 2 edge:  Data(x=[2, 3], edge_index=[2, 2], edge_attr=[2, 3], y=[1])
graph with 2 edge, to undirected:  Data(x=[2, 3], edge_index=[2, 4], edge_attr=[4, 3], y=[1])
```
It is recommended to add edge attributes to the input of the ToUndirected() function to explicitly specify which fields in the graph are edge attributes

### Environment

* PyG version: 2.0.3
* PyTorch version: 1.8.0
* OS: Ubuntu 16.04.7 LTS
* Python version: 3.7.11
* CUDA/cuDNN version: 10.2
* How you installed PyTorch and PyG (`conda`, `pip`, source): conda and pip
* Any other relevant information (*e.g.*, version of `torch-scatter`):

**Comment by rusty1s** [comment]
Thanks for reporting. This should be fixed once https://github.com/pyg-team/pytorch_geometric/pull/3880 lands.
